### PR TITLE
Transformations Section Update

### PR DESCRIPTION
**Pull request by thkwznk**

### CODE REVIEW

**Comment by dacap** [comment]
Hi @thkwznk, a PR should avoid to merge other branches (you can rebase your branch and force push in your own branch). Anyway I'll remove the write access just in case (it's too risky for rebase/force pushes). So you can fork the docs repository in your own user and send PRs from there.

P.S. I've just published the rotate page: http://aseprite.localhost/docs/rotate/

**Comment by thkwznk** [comment]
@dacap Sure, I'll fork the repository, cherry-pick the changes to a new branch and continue from there. Any feedback on the content itself?

**Comment by dacap** [comment]
Thanks @thkwznk, I'll review it right now, some general recommendations would be to avoid changing the style of italic text (we use `*this*` instead of `_this_`). Now I'll make other comments inline.

File: canvas.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # Canvas Size
 
 The canvas is the visible area of the sprite. You can change the
-canvas size using *Sprite > Canvas Size...* menu.
+canvas size using _Sprite > Canvas Size..._ menu.
+
+<!-- PREVIEW: GIF, change canvas size to a bigger one, with previously placed sprite that is only partially visible in the original, small canvas and being shown in full after the change -->
```
Comment (dacap) [inline canvas.md:6]: Our current Markdown parser (and old version of Parsedown.php) doesn't support HTML comments, but I'll try to add support to ignore these comments when we parse, they are useful to mark missing parts of the docs 👍

File: rotate-canvas.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Rotate Canvas
+
+You can rotate the [canvas](canvas.md) using _Sprite > Rotate Canvas_ menu. The available options are:
```
Comment (dacap) [inline rotate-canvas.md:3]: Maybe a minor change:
```suggestion
You can rotate the [whole canvas](canvas.md) using *Sprite > Rotate Canvas* menu. The available options are:
```

File: transformations.md [hunk 3]
```diff
@@ -1,12 +1,31 @@
...
+
+You can apply all fundamental transformation to the canvas using options under the _Sprite_ menu:
+
+- [Flip canvas](flip-canvas.md)
+- [Resize, Crop or Trim canvas](canvas.md)
+- [Rotate canvas](rotate-canvas.md)
+
+## Transforming Sprite or Selection
+
+Sprite or a selection can be transformed using options mostly found under the _Edit_ menu, or by using selection handles:
```
Comment (dacap) [inline transformations.md:15]:
```suggestion
A sprite or a selection can be transformed using options mostly found under the *Edit* menu, or by using the selection handles:
```

File: transformations.md [hunk 4]
```diff
@@ -1,12 +1,31 @@
...
-* [Flip selection](flip.md)
-* [Change canvas size](canvas.md)
-* [Move selection](move-selection.md) or [move complete layer/cel](move-tool.md)
-* [Resize selection](resize.md)
-* [Rotate selection](rotate.md)
+## Transforming Canvas
+
+You can apply all fundamental transformation to the canvas using options under the _Sprite_ menu:
+
+- [Flip canvas](flip-canvas.md)
```
Comment (dacap) [inline transformations.md:9]: Prefer using `*` for bulleted lists.
```suggestion
* [Flip canvas](flip-canvas.md)
```
etc.

File: transformations.md [hunk 5]
```diff
@@ -1,12 +1,31 @@
...
+
+- [Flip sprite or selection](flip.md)
+- [Move complete layer/cel](move-tool.md) or [move selection](move-selection.md)
+- [Resize sprite or selection](resize.md)
+- [Rotate sprite or selection](rotate.md)
+
+## Transforming Multiple Layers, Frames or Cels
+
+Transformations will be applied to every layer, frame, and cel selected on the timeline. If no there's no selection on the timeline, transformations will be applied only to the active cel.
+
```
Comment (dacap) [inline transformations.md:25]:
```suggestion
Transformations will be applied to every layer, frame, and cel selected on the [timeline](timeline.md). If there's no selection on the timeline, transformations will be applied only to the active cel.

```

**Review by dacap — CHANGES_REQUESTED** [review]
So far so good @thkwznk 🙏 some minor changes. It's nice to see these sections evolving. Are the most abandoned parts of the documentation.

File: canvas.md (hunk 1 above)
Comment (thkwznk) [reply canvas.md:6]: I remember @martincapello mentioning this in my last PR. I leave these only as markers in the draft for the parts that I need to fill in later - I'll either replace them with images or turn them into separate issues before submitting a final PR.

**Comment by thkwznk** [comment]
> ...some general recommendations would be to avoid changing the style of italic text (we use \*this\* instead of \_this\_).

I use automatic formatting a lot, so I'll set it up to use the \* for italics and bullet lists. 👍

File: rotate-canvas.md (hunk 2 above)
Comment (thkwznk) [reply rotate-canvas.md:3]: Fixed on the fork in [this commit](https://github.com/thkwznk/aseprite-docs/commit/360a6a13688dd9ad92d844728293c8aad9df7a48).

File: transformations.md (hunk 4 above)
Comment (thkwznk) [reply transformations.md:9]: Fixed on the fork in [this commit](https://github.com/thkwznk/aseprite-docs/commit/360a6a13688dd9ad92d844728293c8aad9df7a48).

File: transformations.md (hunk 3 above)
Comment (thkwznk) [reply transformations.md:15]: Fixed on the fork in [this commit](https://github.com/thkwznk/aseprite-docs/commit/360a6a13688dd9ad92d844728293c8aad9df7a48).

File: transformations.md (hunk 5 above)
Comment (thkwznk) [reply transformations.md:25]: Fixed on the fork in [this commit](https://github.com/thkwznk/aseprite-docs/commit/360a6a13688dd9ad92d844728293c8aad9df7a48).

**Comment by thkwznk** [comment]
I'm closing this one and will reopen a final one with visuals from the forked repository.

**Pull request closed without merging**